pkg/set/set.go

```diff
@@ -13,3 +13,10 @@ func SetProfile(ctx context.Context, clientsets *k8sutil.Clientsets, recoveryOpt
 
 	exec.RunCommandInOperatorPod(ctx, clientsets, "ceph", cephArgs, operatorNamespace, storageClusterNamespace, true, false)
 }
+
+func SetLogLevel(ctx context.Context, clientsets *k8sutil.Clientsets, operatorNamespace, storageClusterNamespace string, args []string) {
+
+	cephArgs := []string{"config", "set", args[0], args[1], args[2]}
```

Comment:
We'll need some validation on these arguments so we can ensure it's only being used by the user to set log levels. Otherwise, we have just exposed a general `ceph config set` for any setting, under the alias of `log-level`. :)